tests/acceptance/apps/vhost-new-test.js

```diff
@@ -287,10 +287,6 @@ test(`visit ${appVhostsNewUrl} and create vhost with existing certificates`, fun
 
   signInAndVisit(appVhostsNewUrl);
 
-  stubRequest('post', '/accounts/stubbed-stack/certificates', function() {
-    assert.ok(false, 'should not create a new certificate');
-  });
-
```

Comment:
Was this causing failures?  With this test case, we shouldn't be creating a new certificate.